hash/pc98.xml

```diff
@@ -3505,7 +3505,7 @@ TODO:
 
 <!-- is this hacked as well? if so it shall be removed -->
 	<software name="advlanda" cloneof="advland">
-		<description>Adventureland (Alt)</description>
+		<description>Adventureland (alt)</description>
```

Comment:
This is technically a "2HD unofficial" version. 
The comment above stating "shall be removed" pisses me off, we don't have enough control over Jp 80s computer scene to dictate what is or isn't "legit" for inclusion in the SW list, as long as it eases cataloguing and regtesting is good for me imho.

Reply:
> 

Removed the comment

Reply:
It should still have some kind of comment indicating it’s an unofficial conversion to high density.

Reply:
Added `notes` with this information